runner/orchestration/gateways/local_gateway.ts

```diff
@@ -109,5 +148,9 @@ export class LocalGateway implements Gateway<LocalEnvironment> {
     return this.llm.hasBuiltInRepairLoop === false;
   }
 
+  shouldRetryFailedTests(): boolean {
```

Comment:
I want to make sure we are clear about code execution failures vs. potential test generation repairs

```suggestion
  shouldRetryFailedTestExecution(): boolean {
```